client/state/comments/actions.js

```diff
@@ -0,0 +1,381 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	COMMENTS_RECEIVE,
+	COMMENTS_COUNT_RECEIVE,
+	COMMENTS_REQUEST,
+	COMMENTS_REQUEST_SUCCESS,
+	COMMENTS_REQUEST_FAILURE,
+	COMMENTS_REMOVE_COMMENT,
+	COMMENTS_ERROR_COMMENT
+} from '../action-types';
+import {
+	createCommentTargetId,
+	createRequestId,
+	createUrlFromTemplate,
+	fieldsMapper
+} from './utils';
+
+const MAX_NUMBER_OF_COMMENTS_PER_FETCH = 50;
+const MAX_NUMBER_OF_COMMENTS_IN_FETCH_RESULT = 100; /* https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/posts/%24post_ID/replies/ */
+
+/***
+ * Internal handler for comments request success
+ * @param {Function} dispatch redux dispatch function
+ * @param {String} requestId request identifier
+ * @param {Number} siteId site identifier
+ * @param {Number} postId post identifier
+ * @param {Object[]} comments comments array
+ * @param {Number} totalCommentsCount comments count that the server have on that post
+ */
+function commentsRequestSuccess( dispatch, requestId, siteId, postId, comments, totalCommentsCount ) {
```

Comment:
I've [refactored](https://github.com/Automattic/wp-calypso/pull/3143/files#diff-a1e5174c2173e0927694ec3bd26d79d4R39) it to pass an options object instead.